latex/content/debug.tex

```diff
@@ -1,6 +1,6 @@
 \subsection{Debug}
 
-\begin{frame}[fragile]{\lstinline+\\overfullrule+}
+\begin{frame}[fragile]{\texttt{\textbackslash overfullrule}}
```

Comment:
It's hard to read in the slide title, because its red on grey.
White typewriter is much better for the title